packages/@okta/vuepress-site/docs/guides/add-an-external-idp/create-app-in-okta/index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Create an App in Okta
+---
+You need to create an OpenID Connect app in your Okta org. This app consumes the response from the Identity Provider after authentication and authorization, allowing user access to your application. Users that sign in for the first time are created in Okta and are associated with this application.
```

Comment:
I would try to simplify the language in the 2nd sentence: `This app consumes the response...` Can we boil this down to the why/purpose of the app vs. what the app does? The two concepts are very similar, but how you describe them are slightly different. For example, `You need to create this app in order to define where the user gets routed to after the Identity Provider authenticates and authorizes the user.`